python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/asset_decorator.py

```diff
@@ -99,6 +99,7 @@ def github_reactions_dagster_assets(context: AssetExecutionContext, dlt: DltDags
                     META_KEY_SOURCE: dlt_source,
                     META_KEY_PIPELINE: dlt_pipeline,
                     META_KEY_TRANSLATOR: dlt_dagster_translator,
+                    **(dlt_dagster_translator.get_metadata(dlt_source_resource) or {}),
```

Comment:
`or {}` is unnecessary

```suggestion
                    **dlt_dagster_translator.get_metadata(dlt_source_resource),
```